.github/release-config.json

```diff
@@ -20,6 +20,14 @@
         "control-plane/agpctl": {
             "package-name": "agpctl",
             "release-type": "go"
+        },
+        "data-plane/integrations/mcp/agp-mcp/examples/llamaindex-time-agent": {
+            "package-name": "agp-llamaindex-time-agent",
+            "release-type": "simple"
```

Comment:
Shouldn't this be python?

Reply:
We want to release docker image and not python